toolkit/docs/quick_start/quickstart.md

```diff
@@ -0,0 +1,128 @@
+
+**Build a VHD or VHDX Image**
+```bash
+# Switch to the toolkit folder 
+cd toolkit
+
+# Build VHDX Image
+# Image is placed in ../out/images/core-efi
+sudo make image REBUILD_TOOLS=y REBUILD_PACKAGES=n CONFIG_FILE=./imageconfigs/core-efi.json
+
+# Build VHD Image
+# Image is placed in ../out/images/core-legacy
+sudo make image REBUILD_TOOLS=y REBUILD_PACKAGES=n CONFIG_FILE=./imageconfigs/core-legacy.json
+```
+
+**Build the cloud-init configuration image**
+```Bash
+# Build the cloud-init configuration image 
+# The output image is ../out/images/meta-user-data.iso
+sudo make meta-user-data
+
+```
+
+**Copy VHD(X) and ISO Images to Your VM Host Machine**
+
+**Create VHD(X) Virtual Machine with Hyper-V**
+
+1. From Hyper-V Select _Action->New->Virtual Machine_.
+1. Provide a name for your VM and press _Next >_.
+1. For VHD select _Generation 1_, For VHDX select _Generation 2_, then press _Next >_.
+1. Change Memory size if desired, then press _Next >_.
+1. Select a virtual switch, then press _Next >_.
+1. Select _Use an existing virtual hard disk_, then browse and select your VHD(X) file.
+1. Press _Finish_.
+
+**Mount the Meta-User-Data.Iso Image**
+
+1. Right click your virtual machine from Hyper-V Manager
+1. Select _Settings..._.
+1. For Gen2/VHDX Images only, select Security and disable _Enable Secure Boot_.
+1. For Gen2/VHDX Images only, select the SCSI Controller from the Hardware panel, choose DVD Drive and press Add.
+1. Select the _DVD Drive_. For Gen1/VHD Images, this is nested under _IDE Controller 1_. For Gen2/VHDX Images, this is nested under _SCSI Controller_.
+1. Select _Image File:_ and browse to the meta-user-data.iso file.
+1. Select _Apply_ to apply all changes.
+
+**Boot and Sign-In to your VHD**
+
+1. Right click your VM and select _Connect..._.
+1. Select _Start_.
+1. Wait for CBL-Mariner to boot to the login prompt, then sign in with:
+
+       mariner_user
+       p@ssw0rd
+
+
+### ISO Image
+
+The following builds a bootable ISO image from precompiled RPMs in the CBL-Mariner package repository at https://packages.microsoft.com/cbl-mariner/1.0/prod/.
+
+```bash
+# Switch to the toolkit folder 
+cd toolkit
+
+# Image is placed in ../out/images/full
+sudo make iso REBUILD_TOOLS=y REBUILD_PACKAGES=n CONFIG_FILE=./imageconfigs/full.json
+```
+**Copy VISO Image to Your VM Host Machine**
+
+**Create VHD(X) Virtual Machine with Hyper-V**
+
+1. From Hyper-V Select _Action->New->Virtual Machine_.
+1. Provide a name for your VM and press _Next >_.
+1. Select _Generation 1_ (VHD) or _Generation 2_ (VHDX), then press _Next >_.
+1. Change Memory size if desired, then press _Next >_.
+1. Select a virtual switch, then press _Next >_.
+1. Select _Create a virtual hard disk_, choose a location for your VHD(X) and set your desired disk Size.  Then press _Next >_.
```

Comment:
>desired disk Size [](start = 87, length = 17)

Might be reasonable to suggest a minimal size here. Not sure what that was for ISO, though, but let's say 4 GBs for safety?